src/sentry/static/sentry/app/views/settings/projectPlugins/details.jsx

```diff
@@ -154,17 +152,9 @@ class ProjectPluginDetails extends AsyncView {
   renderActions() {
     const {pluginDetails} = this.state;
     const enabled = this.getEnabled();
-    const {pluginId} = this.props.params;
-
-    const disabled = DEPRECATED_PLUGINS.includes(pluginId);
 
     const enable = (
-      <Button
-        disabled={disabled}
-        size="small"
-        onClick={this.handleEnable}
-        style={{marginRight: '6px'}}
-      >
+      <Button size="small" onClick={this.handleEnable} style={{marginRight: '6px'}}>
```

Comment:
NIT: We could use a `space` util here.

Reply:
and emotion 🙃

Reply:
@matejminar @priscilawebdev good suggestions, I'll remember to make those changes when I convert this file in the future.